tests/test_wave_01.py

```diff
@@ -51,23 +51,21 @@ def test_get_task(client, one_task):
     }
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+#@pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task_not_found(client):
     # Act
-    response = client.get("/tasks/1")
+    task_id = "1"
+    response = client.get(f"/tasks/{task_id}")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {"message": f"{task_id} not found"}
 
-    raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
 
-
-@pytest.mark.skip(reason="No way to test this feature yet")
 def test_create_task(client):
+    # TODO revert task id 3 to 1
+    task_id = 1
```

Comment:
This is a reasonable thing to do in tests by the way!

For example, you could do it with all the test data like this, and putting it in constants:

```python
def test_create_task(client):
     # Arrange
     EXPECTED_TASK_ID = 1
     EXPECTED_TITLE = "A Brand New Task"
     EXPECTED_DESCRIPTION = "Test Description

     # Act
     response = client.post("/tasks", json={
          "title": EXPECTED_TITLE,
          "description": EXPECTED_DESCRIPTION,
      })
      response_body = response.get_json()

      # Assert
      assert response.status_code == 201
      assert "task" in response_body
      assert response_body == {
        "task": {
            "id": EXPECTED_TASK_ID,
            "title": EXPECTED_TITLE,
            "description": EXPECTED_DESCRIPTION,
            "is_complete": False
          }
      }
```

Well, you could if you wrote the test. I know we wrote this test, but I just wanted to point this pattern out.